modules/nwtc-library/src/NWTC_Base.f90

```diff
@@ -58,16 +58,17 @@ MODULE NWTC_Base
    
    INTEGER(IntKi), PARAMETER     :: NWTC_MAX_DLL_PROC  = 5                        !< maximum number of procedures that can be dynamically loaded from a DLL (see DLL_Type nwtc_base::dll_type)
    
+   TYPE(C_FUNPTR), PARAMETER     :: NULL_PROC_ADDR(NWTC_MAX_DLL_PROC) = C_NULL_FUNPTR  !< this is a hack so the Flang compiler will initialize ProcAddr to C_NULL_FUNPTR in DLL_Type (remove if no longer needed)
```

Comment:
Creating a array of null function pointers seemed to be the only way for Flang to allow initialization of `ProcAddr` in `DLL_Type`. This will provide the same initialization as before, it just required an additional line of code.

Reply:
That's elegant!

Reply:
After merging, I discovered this line causes a seg-fault with gfortran 10.4.0